src/nitypes/scalar.py

```diff
@@ -46,6 +46,79 @@ class Scalar(Generic[TScalar_co]):
     >>> Scalar("value", "volts")
     nitypes.scalar.Scalar(value='value', units='volts')
 
+    Comparing Scalar Objects
+    ^^^^^^^^^^^^^^^^^^^^^^^^
+
+    You can compare scalar objects of compatible types and with identical units
+    using the standard comparison operators (`<`, `<=`, `>`, `>=`, `==`, and `!=`).
```

Comment:
I don't think these operators are rendering the way you expected:
<img width="561" height="450" alt="image" src="https://github.com/user-attachments/assets/2cf7e527-7356-4755-95b7-d0a3a24caadb" />

In reStructuredText, single backtick does not mean code like it does in Markdown: 
https://docutils.sourceforge.io/docs/user/rst/quickref.html#contents

Here's an example of double backticks around operators in the Python docs: https://github.com/python/cpython/blob/main/Doc/reference/expressions.rst?plain=1#L1610